Doc/library/os.rst

```diff
@@ -226,12 +222,13 @@ process and user.
    :data:`os.environ`, and when one of the :meth:`pop` or :meth:`clear` methods is
    called.
 
+   .. seealso::
+
+      The :func:`~os.reload_environ` function.
```

Comment:
```suggestion
      The :func:`os.reload_environ` function.
```

On this page, it seems that we usually add the `os` prefix for clarity, so I would also keep it here.